test/distributed/org/apache/cassandra/distributed/impl/IsolatedJmx.java

```diff
@@ -18,38 +18,40 @@
 
 package org.apache.cassandra.distributed.impl;
 
-import java.lang.reflect.Field;
+import java.io.IOException;
 import java.net.InetAddress;
-import java.net.MalformedURLException;
 import java.util.HashMap;
-import java.util.Iterator;
+import java.util.LinkedList;
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
 
 import javax.management.remote.JMXConnector;
-import javax.management.remote.JMXConnectorFactory;
 import javax.management.remote.JMXConnectorServer;
 import javax.management.remote.JMXServiceURL;
 import javax.management.remote.rmi.RMIConnectorServer;
 import javax.management.remote.rmi.RMIJRMPServerImpl;
 
 import org.slf4j.Logger;
 
+import com.google.monitoring.runtime.instrumentation.common.util.concurrent.Uninterruptibles;
```

Comment:
Should this be `com.google.common.util.concurrent.Uninterruptibles;`?

Reply:
Yup - hit Option->Enter to quickly it seems. Fixed.